lib/reactotron-react-js/package.json

```
@@ -0,0 +1,113 @@
+{
+  "name": "reactotron-react-js",
+  "version": "2.1.1",
+  "description": "A development tool to explore, inspect, and diagnose your React JS/DOM apps.",
+  "author": "Infinite Red",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/infinitered/reactotron/issues"
+  },
+  "homepage": "https://github.com/infinitered/reactotron-react-js",
+  "repository": "https://github.com/infinitered/reactotron-react-js",
```

Small change, non-blocking
```suggestion
  "homepage": "https://github.com/infinitered/reactotron/tree/alpha/lib/reactotron-react-js",
  "repository": "https://github.com/infinitered/reactotron/tree/alpha/lib/reactotron-react-js",
```